cl/search/templates/includes/search_result.html

```diff
@@ -27,7 +27,9 @@ <h3 class="bottom serif">
       <a href="{{ result.absolute_url }}?{{ request.META.QUERY_STRING }}"
          class="visitable">
       {{ result.solr_highlights.caseName.0|safe }}
+      {% if request.path != "/court/me/" %}
```

Comment:
We shouldn't hard code a URL. Did you try doing this with template tags?

Reply:
this is a specific fix for only Maine, and I didn't want to implement the fix in the search_results just for Maine because we certainly dont want this to be a downstream effect on all Maine opinions on the regular search page.  

That is why I originally passed in a variable to handle this edge case.

Reply:
like this ?

https://github.com/freelawproject/courtlistener/pull/2449/commits/1cf7e16ece6729d40c92569c59f5e71cb257a605

Reply:
That was my original solution, I believe, but I think we may have gotten mix messaged because of the scotus code.

Reply:
No, just don't hard code the URL. 

So it becomes:

```
{% url 'some_url_name' as maine_homepage %}
{% if request.path != maine_homepage %}
```